src/vue-wrapper.ts

```diff
@@ -109,12 +112,20 @@ export class VueWrapper<T extends ComponentPublicInstance>
     const rootElementWrapper = new DOMWrapper(this.element)
     return rootElementWrapper.trigger(eventString)
   }
+
+  unmount() {
+    if (this.parentElement) {
+      this.parentElement.removeChild(this.element)
```

Comment:
looks good!

I think we should add a test to ensure this removal is happening?

Reply:
done :)